retail/api/grs-product/AddFulfillmentPlaces.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Copyright 2021 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START retail_add_fulfillment_places]
+
+using Google.Cloud.Retail.V2;
+using Google.Protobuf.WellKnownTypes;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+using System;
+using System.Threading;
+
+namespace grs_product
+{
+    public static class AddFulfillmentPlaces
+    {
+        private const string Endpoint = "retail.googleapis.com";
+        private const string ProductId = "add_fulfillment_test_product_id";
```

Comment:
Should we use a random ID, has the same concern that product may not be deleted so the test may fail when creating the project with the same ID.